inbm/proto/inbs-mock/pb/inbs_sb_grpc.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go-grpc. DO NOT EDIT.
 // versions:
-// - protoc-gen-go-grpc v1.3.0
-// - protoc             v3.12.4
+// - protoc-gen-go-grpc v1.4.0
```

Comment:
<h3>$\color{#15F4EE}Miscellaneous$</h3>

Ensure that the updated gRPC service versions are compatible with the system's requirements and dependencies.